projects/admin/src/app/record/detail-view/statistics-cfg-detail-view/statistics-cfg-detail-view.component.html

```diff
@@ -29,34 +29,43 @@ <h1>{{ record.metadata.name }}</h1>
           [ngClass]="{'text-success': record.metadata.is_active, 'text-danger': !record.metadata.is_active}"
         ></i>
       </dd>
+      <dt class="col-3 label-title" translate>Frequency</dt>
+      <dd class="col-9">{{ record.metadata.frequency | translate }}</dd>
+    </dl>
 
-      <dt class="col-3 label-title" translate>Category</dt>
-      <dd class="col-9">{{ record.metadata.category.type | translate }}</dd>
+    <div class="card mt-3">
+      <h5 class="card-header" translate>Configuration</h5>
+      <div class="card-body">
+        <dl class="row mb-0">
```

Comment:
`row` without parent `container` class isn't recommended. Probably than you should add `container | container-fluid` to `card-body` div.

Additionally is that `mb-0` really necessary on `row`. Is the problem is on `row` on or `card-body` ?